### ISSUE TITLE
The colors  of text in the debug console window are contradict with each other

### ISSUE THREAD

**Issue by zhengger** [Dart-Code/Dart-Code#1845]
For example in the screenshot attached below, the "**_Element.updateChild.<anonymous closure>_** " part in the line "**_I/flutter ( 4435): #2      Element.updateChild.<anonymous closure> package:flutter/…/widgets/framework.dart:286_**" is invisible due to its white color which is contradict with the words in the front which is black.
Can you fix it or give me some suggestion. Thanks!

![image](https://user-images.githubusercontent.com/2143234/60918074-f1786600-a2c4-11e9-9f00-35c1c9a0da6e.png)

**Comment by DanTup** [comment]
Which VS Code colour theme are you using? With the default VS Code themes the colours are automatically adjusted so they work on both light and dark. The Solarized theme doesn't seem to do this, but it also doesn't look like yours.

**Comment by zhengger** [comment]
Hi Dan,
Thank you so much.
I'm using [Github Light Theme](https://marketplace.visualstudio.com/items?itemName=Hyzeta.vscode-theme-github-light). 
Besides this theme I also tried other light themes but the problem remained.
Well, I just found a workaround...Just set  "terminal.ansiWhite": "#450af7", and then the invisible text appeared in "#450af7" color.

![image](https://user-images.githubusercontent.com/2143234/60993368-ed118300-a380-11e9-9ba1-716f86437fd0.png)

**Comment by DanTup** [comment]
Looking at the built-in defaults, it seems like it's expected that light themes will set `ansiWhite` to something dark:

https://github.com/microsoft/vscode/blob/4330dd0eb41c2a2b5773ce2f05d1160f0b39bc76/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts#L97

```js
	'terminal.ansiWhite': {
		index: 7,
		defaults: {
			light: '#555555',
			dark: '#e5e5e5',
			hc: '#e5e5e5'
		}
	},
```

But weirdly, the same is not done for black:

```js
	'terminal.ansiBlack': {
		index: 0,
		defaults: {
			light: '#000000',
			dark: '#000000',
			hc: '#000000'
		}
	},
```

So it's not clear how this is expected to work! I'll see if I can come up with something that behaves better in both (like a grey).

**Comment by DanTup** [comment]
Thanks - I've changed to using 256 bit colours and selected a medium grey which seems to work ok on all themes I tried (including the ones you linked). FWIW, the frames being greyed are framework frames, so hopefully the ones you're less interested in (it was done so your own code stands out in the stack trace). There are some upcoming changes to how errors are shown though, so this may change further in an upcoming release.
